Description: Upstream changes introduced in version 0.9-1~nattyubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 bumblebee-ui (0.9-1~nattyubuntu1) natty; urgency=high
 .
   * Initial Release..
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Juhl (MrMEEE) <mj@casalogic.dk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- bumblebee-ui-0.9.orig/app/Config.py
+++ bumblebee-ui-0.9/app/Config.py
@@ -40,7 +40,7 @@ user_desktop_file_directory = user_home_
 global_desktop_file_directory = '/usr/share/applications/'
 
 #ICONS FILE PATH
-icon_file_directory = '/usr/local/bin/icons/'
+icon_file_directory = '/usr/share/bumblebee-ui/icons/'
 
 #CONFIG FILES PATH
 ui_config_file_directory = user_desktop_file_directory
